logprep/processor/processor_registry.py

```diff
@@ -27,7 +27,7 @@ class ProcessorRegistry:
     mapping = {
         "clusterer": Clusterer,
         "datetime_extractor": DatetimeExtractor,
-        "delete": Delete,
+        "deleter": Deleter,
```

Comment:
This might be slightly hacky but would adding an extra mapping for the delete class prevent the breaking change?

```suggestion
        "deleter": Deleter,
        "delete": Deleter,
```

Reply:
Well we could do it the same way with the HMAC-Option. Meaning we can introduce the inconsistency in the mapping while at the same time marking the `delete` processor as deprecated, which will then be removed in the next mayor release. That way we can now introduce the change, but have one mayor update with all the breaking changes included.

Reply:
lets do it

Reply:
agreed. however, we also need a changelog for this PR.